tests/scripts/fetch_target_logs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x
```

Comment:
Why not `set -eux`? `-eu` is safeguarding against many hidden errors, failing early rather than late.

Reply:
I think we don't want to fail the CI run because of the log collection error. That is why we use only `-x`.

Reply:
Yes, true.